File: containerLoop/4.py
in_str = input("이름을 입력해주세요.\n")

members = ['egoing', 'k8805']

for member in members:
    if member == in_str:
        print("Hello, " + member)
        break
        # import sys
        # sys.exit()
    else:
        print('Who are you?')
        break

File: containerLoop/4.rb
puts("이름을 입력해주세요.\n")
in_str = gets.chomp()

members = ['egoing', 'k8805']

for member in members do
    if member == in_str
        puts("Hello, " + member)
        break
        # import sys
        # sys.exit()
    else
        puts('Who are you?')
        break
    end
end